src/modules/updateTask.js

```diff
@@ -0,0 +1,12 @@
+// update task when it is edited
+const updateTask = (id, description) => {
+  const getTasks = JSON.parse(window.localStorage.getItem('tasks')) || [];
+  getTasks.forEach((object) => {
+    if (object.index === id) {
+      object.description = description;
+    }
```

Comment:
- [optional] Great job making the change. Please ensure that when editing a task, the delete icon stay in its original position rather than coming beside the edited text. Nice job.